### PR TITLE
add "Tours - Orthophotos 2013" (Tours, 37, FR)

### PR DESCRIPTION
**Pull request by Cyrille37**

### CODE REVIEW

File: sources/europe/Tours-Orthophoto-2013.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
+    "url": "http://wms.openstreetmap.fr/tms/1.0.0/tours_2013/{zoom}/{x}/{y}", 
+    "attribution": {
+        "url": "http://wiki.openstreetmap.org/wiki/Tours/Orthophoto", 
+        "text": "Orthophoto Tour(s)plus 2013", 
+        "required": true
+    }, 
+    "type": "tms",
+    "name": "Tours - Orthophotos 2013",
+    "start_date": "2008",
```
Comment (pnorman) [inline sources/europe/Tours-Orthophoto-2013.json]: Is there imagery in this source that goes all the way back to 2008?

File: sources/europe/Tours-Orthophoto-2013.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    "end_date": "2013",
+    "country_code": "FR",
+    "extent": {
+        "max_zoom": 22,
```
Comment (pnorman) [inline sources/europe/Tours-Orthophoto-2013.json]: This seems high - 2cm/px or so. Is the imagery really that high?

**Comment by pnorman** [comment]
please add a `license_url`

File: sources/europe/Tours-Orthophoto-2013.json (hunk 2 above)
Comment (Cyrille37) [inline sources/europe/Tours-Orthophoto-2013.json]: the zoom 22 is on the tms server, you can have a look at http://wms.openstreetmap.fr/tms/1.0.0/tours_2013/. I did not compute tiles, I don't know more. Sorry

**Pull request merged**